ClientPlugin/Config.cs

```diff
@@ -0,0 +1,57 @@
+using ClientPlugin.Settings.Elements;
+using Sandbox.Graphics.GUI;
+using System;
+using System.Text;
+using System.Xml.Serialization;
+using VRage.Input;
+using VRageMath;
+
+
+namespace ClientPlugin.Settings
```

Comment:
Namespace should be just `ClientPlugin`

Reply:
Done in https://github.com/sepluginloader/PluginTemplate/pull/22/commits/156af060632153fd3e20c99ca5797d8f48a7c31d